docs/components/Page.js

```diff
@@ -7,11 +7,14 @@ import Toc from './Toc.js';
 type Props = {|
   children: Node,
   title: string,
+  isFullWidth?: boolean,
 |};
 
-export default function Page({ title: page, children }: Props): Node {
+export default function Page({ title: page, children, isFullWidth = false }: Props): Node {
   const sections = Children.toArray(children);
 
+  const DETAIL_PAGE_MAX_WIDTH = 1312;
```

Comment:
nittest of nits: these kind of consts are easier to find when at the top of a file, just below the imports